proto/expansion.proto

```diff
@@ -24,9 +24,11 @@ message Expansion {
   repeated uint32 durations = 2 [packed=true];
   repeated uint32 distances = 3 [packed=true];
   repeated EdgeStatus edge_status = 4;
-  repeated uint32 edge_id = 5 [packed=true];
-  repeated uint32 pred_edge_id = 6 [packed=true];
+  // deprecated = 5 
+  // deprecated = 6 
   repeated ExpansionType expansion_type = 8;
+  repeated uint64 edge_id = 9 [packed=true];
+  repeated uint64 pred_edge_id = 10 [packed=true];
```

Comment:
I had to go look, but we don't need to deprecate here, `uint32_t` & `uint64_t` are wire-compatible (fwd & bwd): https://protobuf.dev/programming-guides/proto3/#conditionally-safe-changes.

> int32, uint32, int64, uint64, and bool are all compatible.
> If a number is parsed from the wire which doesn’t fit in the corresponding type, you will get the same effect as if you had cast the number to that type in C++ (for example, if a 64-bit number is read as an int32, it will be truncated to 32 bits).